examples/tensorflow_estimator_simple.py

```diff
@@ -40,25 +40,25 @@ def create_network(trial, features):
     n_layers = trial.suggest_int('n_layers', 1, 3)
     for i in range(n_layers):
         n_units = trial.suggest_int('n_units_l{}'.format(i), 1, 128)
-        prev_layer = tf.layers.dense(inputs=prev_layer, units=n_units, activation=tf.nn.relu)
+        prev_layer = tf.compat.v1.layers.dense(
```

Comment:
The reference of `tf.compat.v1.layers.dense` says that it is deprecated and will be removed in the future. Can we replace it with the up-to-date method? Or do we keep it as it is for now?
https://www.tensorflow.org/api_docs/python/tf/compat/v1/layers/dense

> Warning: THIS FUNCTION IS DEPRECATED. It will be removed in a future version. Instructions for updating: Use keras.layers.Dense instead.

Reply:
@toshihikoyanase 

Thanks, I missed this message.
I used `tf.keras.layers.Dense` instead.

Could you check it?